packages/components/src/FilterBar/FilterBar.component.js

```diff
@@ -79,9 +87,9 @@ FilterInput.propTypes = {
 	debounceMinLength: PropTypes.number,
 	debounceTimeout: PropTypes.number,
 	onBlur: PropTypes.func,
+	onClear: PropTypes.func,
 	onFocus: PropTypes.func,
 	onFilter: PropTypes.func.isRequired,
-	onToggle: PropTypes.func,
```

Comment:
It looks like a breaking change, could you update the PR description please ?

Reply:
discussed this is only in an internal component so no breaking changes